project-reports/2023/2023-Q2-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,105 @@
+
+Topology rotation was rewritten:
+- Previously, it was possible for the leader to manipulate the block and affect the topology in the next consensus round
+- Topology is not calculated based on hash of the current block anymore, it's determined from the previous topology and the view change index
+
+The `iroha_client_cli` output is now machine readable, leading to improved tests.
+
+The dependency on the unstable feature `default_alloc_error_handler` is not needed anymore in WASM smart contracts. It was fixed by using the stable default allocator handler ([#3303](https://github.com/hyperledger/iroha/pull/3303)) [^1]:
+
+- Iroha doesn't depend on any unstable features anymore
+- `data_model` structs are now opaque to the client / `WASM` code
+- This was an effort to maximize portability and maintain the same API for client apps and `WASM` smartcontracts
+- All fields have been made opaque and can be accessed through getter functions
+
+The Iroha 2 schema was refactored:
+
+- Every name in the codebase is globally unique
+- Prefixes from type names have been removed
+- The schema file is now sorted lexicographically
+
+Legacy configurations have been added to Iroha. ([#3183](https://github.com/hyperledger/iroha/commit/ac32c4934d43ef85445a5e9c70711f1582be3269))
+
+The `AssetDefinition` now has a logo field. ([#3389](https://github.com/hyperledger/iroha/pull/3389))
+
+Iroha 2 now uses Tokio actors. This allows us to use industry standard instrumentation to diagnose deadlocks, starvation  and other previously undiagnoseable problems. ([#3362](https://github.com/hyperledger/iroha/issues/3362))
+
+Additionally an SDK based off of using `iroha_client_cli` and a common JSON API was proposed, and will be merged.
+
+### Documentation updates
+
+We have conducted a revision of the original Iroha v2 tutorial to identify issues and outdated content. We have onboarded two new technical writers and have constructed a roadmap of advanced explanations. Work is underway for addition of detailed iroha_client_cli documentation as well as additional elements regarding architectural design. We plan to expand the tutorial with a section that was used as a formal reports for one of our internal projects.
+
+### Mentorships
+
+There are three mentorship streams currently accepting mentees for Iroha 2.
+
+* [Hyperledger - Iroha 2 FFI client library bindings](https://mentorship.lfx.linuxfoundation.org/project/8a6b5853-369c-48e6-9445-98520a8c28dc)
+* [Hyperledger - Iroha 2 blockchain explorer update](https://mentorship.lfx.linuxfoundation.org/project/24e1907b-c8e0-488c-aa24-1fbddb9df886)
+* [Hyperledger - Iroha 2 DSL](https://mentorship.lfx.linuxfoundation.org/project/cda1496b-0c36-4c1b-98b5-dee5d95f494a)
+
+# Maintainer Diversity
+
+Currently, the Iroha 2 core development team consists of:
+
+* 8 Rust engineers + 1 tech lead
```

Comment:
are the team members all Soramitsu employees? please mark their employers which is the main aspect for diversity